docs/reference/commandline/volume_create.md

```diff
@@ -29,6 +29,8 @@ The mount is created inside the container's `/src` directory. Docker does not su
 
 Multiple containers can use the same volume in the same time period. This is useful if two containers need access to shared data. For example, if one container writes and the other reads the data.
 
+You cannot create a volume if there is a volume already registered with the same name in other driver. Docker returns the existent volume ID if the volume name conflicts and you don't specify the driver.
```

Comment:
@azurezk Thank you for the contribution.  Couple of changes to text and error message.

---> https://gist.github.com/moxiegirl/77c1f117d3080d1c95c6

Volume names must be unique among drivers.  This means you cannot use the same volume name with two different drivers.  If you attempt this `docker` returns an error:

```
A volume named  %s  already exists with the %s driver. Choose a different volume name.
```

If you specify a volume name already in use on the current driver, Docker assumes you want to re-use the existing volume and does not return an error.